integration/combination/test_api_with_authorizers.py

```diff
@@ -0,0 +1,478 @@
+import requests
+
+from integration.helpers.base_test import BaseTest
+from integration.helpers.deployer.utils.retry import retry
+
+
+class TestApiWithAuthorizers(BaseTest):
+    def test_authorizers_min(self):
+        self.create_and_verify_stack("combination/api_with_authorizers_min")
+        stack_outputs = self.get_stack_outputs()
+
+        rest_api_id = self.get_physical_id_by_type("AWS::ApiGateway::RestApi")
+        apigw_client = self.client_provider.api_client
+
+        authorizers = apigw_client.get_authorizers(restApiId=rest_api_id)["items"]
+        lambda_authorizer_uri = (
+            "arn:aws:apigateway:"
+            + self.my_region
+            + ":lambda:path/2015-03-31/functions/"
+            + stack_outputs["AuthorizerFunctionArn"]
+            + "/invocations"
+        )
+
+        lambda_token_authorizer = get_authorizer_by_name(authorizers, "MyLambdaTokenAuth")
+        self.assertEqual(lambda_token_authorizer["type"], "TOKEN", "lambdaTokenAuthorizer: Type must be TOKEN")
+        self.assertEqual(
+            lambda_token_authorizer["identitySource"],
+            "method.request.header.Authorization",
+            "lambdaTokenAuthorizer: identity source must be method.request.header.Authorization",
+        )
+        self.assertIsNone(
+            lambda_token_authorizer.get("authorizerCredentials"),
+            "lambdaTokenAuthorizer: authorizer credentials must not be set",
+        )
+        self.assertIsNone(
+            lambda_token_authorizer.get("identityValidationExpression"),
+            "lambdaTokenAuthorizer: validation expression must not be set",
+        )
+        self.assertEqual(
+            lambda_token_authorizer["authorizerUri"],
+            lambda_authorizer_uri,
+            "lambdaTokenAuthorizer: authorizer URI must be the Lambda Function Authorizer's URI",
+        )
+        self.assertIsNone(
+            lambda_token_authorizer.get("authorizerResultTtlInSeconds"), "lambdaTokenAuthorizer: TTL must be null"
```

Comment:
I would use the same notation as above: "must not be set"

Reply:
Make sense to me, will fix it.